test_runner/src/main/kotlin/ftl/run/DumpShards.kt

```diff
@@ -35,7 +35,7 @@ fun dumpShards(
 ) {
     saveShardChunks(
         shardFilePath = shardFilePath,
-        shards = args.testShardChunks,
+        shards = args.testShardChunks.map { it.testsList },
```

Comment:
nit: `testShardChunks.map { it.testsList }` method chain is repeated a couple of places, would it be better to have extension function for mappinig testsList?